polls/templates/polls/results.html

```diff
@@ -0,0 +1,49 @@
+{% extends 'global/skeleton.html' %}
+
+{% block pagename %}Visualizza i risultati{% endblock pagename %}
+
+{% block content %}
+
+    <div class="d-flex justify-content-center">
+        <div class="card text-center" >
+            <div class="card-body">
+                <h3 class="card-title text-primary">Benvenuto su SmoothPoll!</h3>
+                <h5 class="card-title text-info">L'applicazione al momento è in costruzione</h5> 
+                <p class="card-text">
+                    Questa è la pagina dei risultati
+                </p>
+                <div class="row">
+                    <div class="col-3">
+                        <p class="text-secondary">Quesito:</p>
+                    </div>
+                    <div class="col-6">
+                        <p class="fw-bold">{{poll.question}}</p>
+                    </div>
+                </div>
+                <div class="container">
+                    {%for option in sorted_options %}
+                        <p>{{option.option.value}}:{{option.n_votes}}</p>
```

Comment:
For now it's okay you did this, but for next time when you have to present list-like stuff consieder different kind of tags. 

You may consieder:
- Simple lists like those: https://www.w3schools.com/html/html_lists.asp 
- More complex Bootstrap grids: https://getbootstrap.com/docs/5.0/layout/grid/